### PR TITLE
Feature/niad 277 fhir validation tool

### PR DESCRIPTION
**Pull request by dawidskainos**

### CODE REVIEW

File: service/src/integration-test/resources/fhir/encounter_valid.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "type": "transaction",
+    "entry": [
+        {
+            "fullUrl": "urn:uuid:14200fa9-00f9-4106-8ce2-11a09c446a93",
+            "resource": {
+                "resourceType": "Encounter",
+                "status": "finished",
+                "serviceProvider": {
+                    "reference": "urn:uuid:23b2c1e7-ebb4-48a0-8e18-1e2716b85752"
+                }
```
Comment (Alicja-Kainos) [inline service/src/integration-test/resources/fhir/encounter_valid.json:12]: Are we supposed to amend this json when adding the mapper bricks to our app, so new elements to our bundle? Do I git it right?

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterReportBundleService.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         Encounter encounter = encounterMapper.mapEncounter();
 
         addEncounter(bundle, encounter);
-        addEpisodeOfCare(bundle, encounter);
+        addServiceProvider(bundle, encounter);
```
Comment (Alicja-Kainos) [inline service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterReportBundleService.java:30]: why here the episode of care has been removed? Adding service provider seems ok.

File: service/src/integration-test/resources/fhir/encounter_valid.json (hunk 1 above)
Comment (dawidskainos) [reply service/src/integration-test/resources/fhir/encounter_valid.json:12]: Until we implement reading from queue - yes. This way we can be sure that we produce a valid FHIR message. Once reading from queue is implemented it'll just work automatically.

File: service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterReportBundleService.java (hunk 2 above)
Comment (dawidskainos) [reply service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterReportBundleService.java:30]: Episode of care is a more complex element and I had troubles creating a valid message - so I replaced it with a simple one which has no mandatory elements.

**Pull request merged**